### PR TITLE
Redis cloud update - v1.1.0 

### PR DESCRIPTION
**Pull request by j8-redis**
### What does this PR do?

Adds three dashboards and makes minor changes to an existing dashboard

### Motivation

Client requests

### Review checklist

check

### Additional Notes

Anything else we should know when reviewing?

No

### CODE REVIEW

File: redis_cloud/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
 # CHANGELOG - Redis Cloud
 
+## 1.1.0 / 2024-11-04
+
+***Changed***:
+
+* Removed redis_cloud.yaml file
+* Changed version to 1.1.0
```
Comment (eho1307) [inline redis_cloud/CHANGELOG.md:8]:
```suggestion
```
not needed

**Review by eho1307 — CHANGES_REQUESTED** [review]

**Comment by rtrieu** [comment]
Created DOCS-9427 for review.

**Review by jjackson-dd — APPROVED** [review]

**Review by eho1307 — APPROVED** [review]

**Review by steveny91 — APPROVED** [review]

File: redis_cloud/assets/dashboards/redis_cloud_proxy-threads.json [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+        "background_color": "vivid_blue",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 7104524980730882,
+            "definition": {
+              "type": "note",
+              "content": "Redis Cloud allows multiple databases to be created. Each database gets an endpoint (a unique URL and port on the FQDN). This endpoint receives all the traffic for all operations for that database. By default, Redis Software binds this database endpoint to one of the proxies on a single node in the cluster. This proxy becomes an active proxy and receives all the operations for the given database. (note that if the node with the active proxy fails, a new proxy on another node takes over as part of the failover process automatically).",
```
[the remark this answers is not in the document]
Comment (buraizu) [reply redis_cloud/assets/dashboards/redis_cloud_proxy-threads.json:18]: Punctuation and capitalization rules change when a parenthetical is a standalone sentence.

**Review by buraizu — APPROVED** [review]
Approving with some minor updates requested

**Review by jjackson-dd — APPROVED** [review]

**Pull request merged**